testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -748,6 +748,10 @@
   end
 end
 
+Then(/^I should not see a warning sign$/) do
+  raise 'Errors detected in the Products Catalog' unless page.has_no_xpath?("//*[contains(@class, 'fa fa-li fa-exclamation-triangle text-warning')]", wait: 5)
```

Comment:
```suggestion
  raise 'Warning detected' unless page.has_no_xpath?("//*[contains(@class, 'fa fa-li fa-exclamation-triangle text-warning')]", wait: 5)
```

We could be using this step in other scenarios where we have warning signs. Let's be generic in the error description.